api_test.go

```diff
@@ -266,6 +284,7 @@ func TestEndNodeStateFail(t *testing.T) {
 	if err == nil {
 		t.Errorf("should fail.")
 	}
+
```

Comment:
Additional empty line should be removed.
Becaue it is not related with this PR.